client/app/lib/util/appendHeadElement.coffee

```diff
@@ -1,7 +1,7 @@
-kd = require 'kd'
-
 module.exports = ({ type, url }, callback) ->
 
+  kd = require 'kd'
+
```

Comment:
hmm, to be able to mock `appendHeadElement` without mocking `kd`, it's probably not required anymore, but not sure.